java/core/src/test/java/co/worklytics/psoxy/RulesTest.java

```diff
@@ -15,15 +15,14 @@
 
     final String EXAMPLE_YAML =
         "pseudonymizations:\n" +
-        "- relativeUrlRegex: \"/calendar/v3/calendars/.*/events.*\"\n" +
-        "  jsonPaths:\n" +
-        "  - \"$..email\"\n" +
-        "emailHeaderPseudonymizations: []\n" +
-        "redactions:\n" +
-        "- relativeUrlRegex: \"/calendar/v3/calendars/.*/events.*\"\n" +
-        "  jsonPaths:\n" +
-        "  - \"$..displayName\"\n" +
-        "  - \"$.items[*].extendedProperties.private\"\n";
+            "- jsonPaths:\n" +
+            "  - \"$..email\"\n" +
+            "  relativeUrlRegex: \"/calendar/v3/calendars/.*/events.*\"\n" +
```

Comment:
Maybe we could add another example with the rule with start prefix in the regex `^/calendar/v3/calendars/.*/events.` to show that we support it